integrating suggested tags banner (#15165)
DataDog/documentation: content/en/metrics/metrics-without-limits.md

### Configuration of tags

> Click on any metric name to open its details sidepanel. Then click **Manage Tags** -> **“Include Tags...”** to configure the tags you’d like to remain as queryable on dashboards and monitors. The tag configuration modal will pre-populate by default with an allowlist of tags that have been actively queried on dashboards, notebooks, monitors, and through API in the past 30 days (colored in blue with an icon). You may include additional tags as well. Before selecting **Save**, an estimated new volume of indexed custom metrics that results from this potential tag configuration is displayed.

Reviewer: ```suggestion
Click on any metric name to open its details sidepanel. Then click **Manage Tags** -> **“Include Tags...”** to configure the tags you’d like to remain as queryable on dashboards and monitors. The tag configuration modal pre-populates by default with an allowlist of tags that have been actively queried on dashboards, notebooks, monitors, and through API in the past 30 days (colored in blue with an icon). You can also include your own additional tags. Before selecting **Save**, an estimated new volume of indexed custom metrics that results from this potential tag configuration is displayed.
```

You can add or remove aggregations at any time with no required Agent or code-level changes. 

> The tag configuration modal will pre-populate with an allowlist of aggregations that have been actively queried on dashboards, notebooks, monitors and through API in the past 30 days (colored in blue with an icon). You may include additional aggregations as well.

Reviewer: ```suggestion
The tag configuration modal pre-populates with an allowlist of aggregations that have been actively queried on dashboards, notebooks, monitors and through API in the past 30 days (colored in blue with an icon). You can also include your own additional aggregations.
```
